logproxy-client/src/test/java/com/oceanbase/clogproxy/client/LogProxyClientTest.java

```diff
@@ -37,8 +39,11 @@ public void testLogProxyClient() {
         config.setStartTimestamp(0L);
         config.setTableWhiteList("sys.test.*");
 
-        LogProxyClient client = new LogProxyClient("127.0.0.1", 2983, config);
-
+        ClientConf clientConf = new ClientConf();
+        clientConf.setVERSION("2.1.1");
+        clientConf.setTransferQueueSize(20);
+        LogProxyClient client = new LogProxyClient("127.0.0.1", 2983, config, clientConf);
+        logger.info(ClientConf.VERSION);
```

Comment:
These conf should be isolated between LogProxyClient objects, we should not pass a conf object with static fields here.